src/components/CardGrid/CardGrid.tsx

```diff
@@ -1,44 +1,43 @@
 import * as React from "react";
 import { classNames } from "../../lib/classNames";
-import { getClassName } from "../../helpers/getClassName";
-import { usePlatform } from "../../hooks/usePlatform";
 import { withAdaptivity, AdaptivityProps } from "../../hoc/withAdaptivity";
 import "./CardGrid.css";
 
 export interface CardGridProps
   extends React.HTMLAttributes<HTMLDivElement>,
     AdaptivityProps {
   size: "s" | "m" | "l";
+  /**
+   * Если true, то вокруг компонента присутствуют стандартные отсупы сверху/снизу и слева/справа
+   */
+  spaced?: boolean;
 }
 
-/**
- * @see https://vkcom.github.io/VKUI/#/CardGrid
- */
-const CardGrid: React.FunctionComponent<CardGridProps> = ({
+const CardGridComponent = ({
   children,
-  size,
+  size = "s",
+  spaced = false,
   sizeX,
   ...restProps
 }: CardGridProps) => {
-  const platform = usePlatform();
-
   return (
     <div
       {...restProps}
       vkuiClass={classNames(
-        getClassName("CardGrid", platform),
+        "CardGrid",
+        spaced && "CardGrid--spaced",
         `CardGrid--${size}`,
-        `CardGrid--sizeX-${sizeX}`
+        `CardGrid--sizeX-${sizeX}` // TODO: v5 новая адаптивность
       )}
     >
       {children}
     </div>
   );
 };
 
-CardGrid.defaultProps = {
-  size: "s",
-};
+/**
+ * @see https://vkcom.github.io/VKUI/#/CardGrid
+ */
+export const CardGrid = withAdaptivity(CardGridComponent, { sizeX: true });
 
-// eslint-disable-next-line import/no-default-export
-export default withAdaptivity(CardGrid, { sizeX: true });
+CardGridComponent.displayName = "CardGrid"; // TODO: v5 remove
```

Comment:
```suggestion
CardGrid.displayName = "CardGrid"; // TODO: v5 remove
```
вот так вот только :)